components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa.Edge.Publisher/src/Engine/WriterGroupMessageSource.cs

```diff
@@ -54,18 +54,18 @@ public class WriterGroupMessageTrigger : IMessageTrigger, IDisposable {
                 true : false;
 
         /// <inheritdoc/>
-        public OpcAuthenticationMode AuthenticationMode =>_subscriptions?.First()?.Subscription?.Connection.User != null ?
+        public OpcAuthenticationMode AuthenticationMode =>_subscriptions?.First()?.Subscription?.Connection?.User != null ?
```

Comment:
Should we use `_subscriptions?.First()?.Subscription?.Connection?.User?.Value` to keep the condition of `OpcAuthenticationMode` symmetric to that of `AuthenticationUsername` ?

Reply:
agreed